src/main/java/adris/altoclef/Settings.java

```diff
@@ -73,6 +73,17 @@ public class Settings implements IFailableConfigFile {
      * If true, will show a timer.
      */
     private boolean showTimer = false;
+    
+
+    /**
+     * If true, will spread items through crafting grid.
+     */
+    private boolean spreadItemsToCraft = false;
+    
+    /*
+     * If true open inventory during crafting in 2x2
+     */
+    private boolean openInvDuringCrafting = true;
```

Comment:
```suggestion
    private boolean openInventoryDuringCrafting = true;
```
adhering to good naming conventions